blocks.md

```diff
@@ -391,4 +392,4 @@ interval = 1800
 Key | Values | Required | Default
 ----|--------|----------|--------
 bytes | weather to use bytes or bits in the display.<br>(true for bytes, false for bits) | No | false
-interval | Update interval in seconds | No | 1800
\ No newline at end of file
+interval | Update interval in seconds | No | 1800
```

Comment:
I think this line is what is causing the merge conflict. Can you remove this change?